components/shared/gallery/Properties.vue

```diff
@@ -35,3 +34,9 @@ export default class Properties extends Vue {
   @Prop({ type: String, default: 'trait_type' }) public fieldKey!: string
 }
 </script>
+<style scoped>
+.attribute-key {
+  min-height: 18px;
+}
+</style>
```

Comment:
oh no... :(
any other solution than a fixed height? 
could you remove scoped?

Reply:
> oh no... :( any other solution than a fixed height? could you remove scoped?
I found no other better solution in this case. Empty string ' ' does not have height.
Why should we remove `scoped`?

Reply:
https://github.com/kodadot/nft-gallery/issues/1348#issuecomment-1189026590

Reply:
DONE!